### PR TITLE
fix: several bugs

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description

Closes # (NO_ISSUE)
https://mask.atlassian.net/browse/MASK-875
https://mask.atlassian.net/browse/MASK-874

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/plugins/CryptoartAI/SNSAdaptor/ArticleTab.tsx [hunk 1]
```diff
@@ -27,13 +31,15 @@ export function ArticleTab(props: ArticleTabProps) {
...
-                    <Link href={asset.value.ossUrl} target="_blank" rel="noopener noreferrer">
-                        <img className={classes.player} src={resourceUrl} alt={asset.value.title} />
-                    </Link>
-                ) : (
-                    <img className={classes.player} src={resourceUrl} alt={asset.value.title} />
-                )}
+                <NFTCardStyledAssetPlayer
+                    url={resourceUrl}
+                    contractAddress=""
+                    tokenId=""
```
Comment (guanbinrui) [inline packages/mask/src/plugins/CryptoartAI/SNSAdaptor/ArticleTab.tsx]:
```suggestion
```

File: packages/mask/src/social-network-adaptor/twitter.com/injection/ProfileTabContent.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {
     searchProfileEmptySelector,
     searchProfileTabPageSelector,
 } from '../utils/selector'
+import { clear } from './ProfileTab'
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/twitter.com/injection/ProfileTabContent.tsx]: Refactor to use a hook.

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/social-network-adaptor/twitter.com/injection/ProfileTabContent.tsx [hunk 3]
```diff
@@ -70,5 +71,6 @@ const useStyles = makeStyles()((theme) => {
 
 export function ProfileTabContentAtTwitter() {
     const { classes } = useStyles()
-    return <ProfileTabContent classes={classes} />
+    const clear = useClear()
+    return <ProfileTabContent classes={classes} clear={clear} />
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/twitter.com/injection/ProfileTabContent.tsx:75]: Use `useClear` in `<ProfileTabContent />`.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**